src/GZip.jl

```diff
@@ -340,8 +340,9 @@ skip(s::GZipStream, n::Integer) =
            s.gz_file, n, SEEK_CUR)!=-1 ||
      error("skip (gzseek) failed")) # Mimick behavior of skip(s::IOStream, n)
 
-position(s::GZipStream) =
-    ccall((_gztell, _zlib), ZFileOffset, (Ptr{Void},), s.gz_file)
+position(s::GZipStream, of_raw::Bool=false) = of_raw ?
```

Comment:
Minor nit: I would simply call this `raw`.  But since it's not really exposed anywhere, I only care enough to write this comment. ;-)